GiniVision/Classes/OpenWithTutorialCollectionHeader.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  OpenWithTutorialCollectionHeader.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 10/24/17.
+//  Copyright © 2017 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+final class OpenWithTutorialCollectionHeader: UICollectionReusableView {
+    
+    let padding:(top: CGFloat, left: CGFloat, bottom: CGFloat, right: CGFloat) = (20, 20, 20, 20)
+    
+    lazy var headerContainer: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .white
+        return view
+    }()
+    
+    lazy var headerTitle: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.textColor = .black
+        let textSize: CGFloat = UIDevice.current.isIpad ? 16 : 14
```

Comment:
This code is everywhere. Can we standardise these? Say that all iPad fonts are 3-4-5 points bigger?

Reply:
I had some properties for each kind of text (title, subtitle) and It had a version for iPad and iPhone. 
But I realized that it wasn't useful at all, since in some places the title was 16, in others 14... I wouldn't like to have a `dimen.xml` file now, or at least for this PR (It would be a different PR since this issue is spread in the whole library).